scripts/variantstore/wdl/GvsCreateVATfromVDS.wdl

```diff
@@ -34,35 +42,69 @@ workflow GvsCreateVATfromVDS {
             ref = reference
     }
 
-    call StripCustomAnnotationsFromSitesOnlyVCF {
+    call Utils.IndexVcf {
         input:
-            input_vcf = RemoveDuplicatesFromSitesOnlyVCF.output_vcf,
-            custom_annotations_header = MakeSubpopulationFilesAndReadSchemaFiles.custom_annotations_template_file,
-            output_vcf_name = "${input_vcf_name}.unannotated.sites_only.vcf",
-            output_custom_annotations_filename = "ac_an_af.tsv"
+            input_vcf = RemoveDuplicatesFromSitesOnlyVCF.output_vcf
     }
 
-    ## Use Nirvana to annotate the sites-only VCF and include the AC/AN/AF calculations as custom annotations
-    call AnnotateVCF {
+    call Utils.SplitIntervals {
         input:
-            input_vcf = StripCustomAnnotationsFromSitesOnlyVCF.output_vcf,
-            output_annotated_file_name = "${input_vcf_name}_annotated",
-            nirvana_data_tar = nirvana_data_directory,
-            custom_annotations_file = StripCustomAnnotationsFromSitesOnlyVCF.output_custom_annotations_file,
+            intervals = interval_list,
+            ref_fasta = reference,
+            ref_fai = reference_index,
+            ref_dict = reference_dict,
+            scatter_count = effective_scatter_count,
+            output_gcs_dir = output_path + "intervals",
+            split_intervals_disk_size_override = split_intervals_disk_size_override,
+            split_intervals_mem_override = split_intervals_mem_override,
     }
 
-    call PrepVtAnnotationJson {
-        input:
-            annotation_json = AnnotateVCF.annotation_json,
-            output_file_suffix = "${input_vcf_name}.json.gz",
-            output_path = output_path,
-    }
+    String sites_only_vcf_basename = basename(basename(input_sites_only_vcf, ".gz"), ".vcf")
```

Comment:
WDL doesn't let you `basename(..., ".vcf.gz")` in one shot?

Reply:
It does, but I think (hope) I'm handling the case where it may be a `.vcf.gz` OR a `.vcf`